ipld.go

```diff
@@ -15,10 +15,6 @@ import (
 
 var _ ipld.NodeGetter = (*crdtNodeGetter)(nil)
 
-func init() {
-	ipld.Register(cid.DagProtobuf, dag.DecodeProtobufBlock)
-}
```

Comment:
Unclear that this is used for anything in this repo. Is there anything I'm missing here?

Reply:
Well it seems it is not really needed. Perhaps it was needed once upon a time, or I just feared the NodeGetter wouldn't be able to parse anything. (I suspect this codec may be registered by default somewhere else?).

Reply:
it used to be registered by go-merkledag, although I'm not sure it's even needed here anymore